### ISSUE TITLE
[Feature Request] Description for CheckList items, checkboxes, radio

### ISSUE THREAD

**Issue by ziming** [Laravel-Backpack/community-forum#213]
# Feature Request

### What's the feature you think Backpack should have?

Seeing is better than explaining. Basically allow specifying description that will be placed below each radio/checkbox label

https://tailwindui.com/components/application-ui/forms/radio-groups

https://tailwindui.com/components/application-ui/forms/checkboxes

### Have you already implemented a prototype solution, for your own project?

No. At it's easiest I guess you can do something like this

```html
<div>
<label>Laravel Backpack</label>
<span>{!! $field['description'] !!}</span>
</div>
```

Or it can go more advanced

### Do you see this as a core feature or an add-on?

Core

**Comment by tabacitu** [comment]
I agree, I've needed this too at some point and rolled my own field type. I don't think this should be added to the current field types though, we should probably create new ones. Which we will anyway once we create new fields for our Livewire components, because we'll be using Tabler and it has a UI for this already: https://preview.tabler.io/form-elements.html

<img width="318" alt="CleanShot 2022-10-03 at 09 03 21@2x" src="https://user-images.githubusercontent.com/1032474/193510719-c82d20e8-7a1c-48b2-8f50-5f7a99ae0b05.png">

So let's postpone this until then.

Cheers!
